core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```
@@ -546,6 +546,10 @@ class WorkspaceService(
           throw t
       }
     )
+    // Delete the workspace PAO in the policy service (TPS)
+    _ <- traceFutureWithParent("deleteTpsPao", ctx) { span =>
+      policyService.deleteWorkspacePao(UUID.fromString(workspace.workspaceId), span)
+    }
```

If we can't delete the PAO, for whatever reason (other than 404), this will stop the workspace-deletion flow right here and leave the workspace half-deleted. Should this swallow PAO-deletion errors - probably should log them - and continue on?

I know we don't swallow errors for many other parts of the workspace-deletion flow, but we also want this flow to be more stable, so I'm raising the question.